packages/mui-lab/package.json

```
@@ -42,6 +42,8 @@
   },
   "peerDependencies": {
     "@mui/material": "^5.0.0",
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```

```suggestion
    "@emotion/react": "^11.5.0",
    "@emotion/styled": "^11.3.0",
```

Let's match the ones in `@mui/material`.